### ISSUE TITLE
404 - ethereumH page not found

### ISSUE THREAD

**Issue by ponarunkumar** [ethereum/homestead-guide#452]
Navigate > http://www.ethdocs.org/en/latest/ethereum-clients/ethereumh/index.html and the link
https://github.com/blockapps/ethereumH needs to be updated.

**Comment by Sudha247** [comment]
Yeah, I too noticed the link is broken. Is that repo [here](https://github.com/jamshidh/ethereum-client-haskell) now?
